lib/logstash-input-tcp_jars.rb

```diff
@@ -1,4 +1,7 @@
-# encoding: utf-8
+# AUTOGENERATED BY THE GRADLE SCRIPT. DO NOT EDIT.
```

Comment:
the generated file no longer contains `require_jar('org.apache.logging.log4j', 'log4j-api', '2.17.0')`
the file could be .gitignore-d like other plugins do.